packages/api/src/util.ts

```diff
@@ -233,6 +235,8 @@ export function getEnv(): BackendEnv {
     reminderTaskHanderUrl: parse('REMINDER_TASK_HANDLER_URL'),
     integrationTaskHandlerUrl: parse('INTEGRATION_TASK_HANDLER_URL'),
     textToSpeechTaskHandlerUrl: parse('TEXT_TO_SPEECH_TASK_HANDLER_URL'),
+    textToSpeechName: parse('TEXT_TO_SPEECH_QUEUE_NAME'),
```

Comment:
I wonder if we can just hardcode these queue names. Doesn't seem like we will ever want to dynamically change the names